README.md

```diff
@@ -677,6 +733,40 @@ lifecycle, for example when temporarily dealing with high volume events like `sc
 
 Be sure to pass the identical arguments used when calling `addEventListener`!
 
+### Mixins
+
+Ember lifeline also provides mixins, which extend the object's methods to include lifeline's functions.
+
+To use any of the above mentioned functions in your component, route or service, simply import and apply one or all of these mixins to your class:
+
+`ContextBoundTasksMixin` for using any of the *Task methods
+`ContextBoundEventListenersMixin` for using addEventListener
+`DisposableMixin` for using registerDisposable and runDisposable
+
+### Testing
+
+Lifeline's entire purpose is to help ensure you've entangled and ultimately disposed of any outstanding async work in your applications. To help ensure this has occurred, and that you don't have any remaining queued async work, a test helper is provided which will assert that all async is disposed of.
+
+To use the helper using the new ember-qunit module syntax:
+
+```js
+// test-helper.js
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupLifelineValidation from 'ember-lifeline/test-support';
+
+module('module', function(hooks) {
+  setupLifelineValidation(hooks);
```

Comment:
This example is perfect, but we should emphasize a bit more that `setupLifelineValidation` should be done _before_ `setupTest` / `setupRenderingTesf` etc